verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -73,6 +74,47 @@ export default class CasesController {
         }
     };
 
+    /** DownloadAsync spawns a worker thread that forwards the request to the data service
+     * and emails the streamed response as a csv attachment. It will only email to the
+     * logged-in user, and it replies immediately with 204. Any error will be reported
+     * in the email, except the error of not being able to email which we can only log.
+     * 
+     * Question for code reviewers: do we want to retry on errors? If so we need to introduce
+     * a message bus rather than just using background workers, and we're probably back at the
+     * point where we use Batch for this.*/
```

Comment:
I don't know if retrying on errors would provide much benefit, i.e. why wouldn't retrying simply run into the same error. That said, as the data set and user base grow, it might make more sense to use some combination of AMQP/SQS and Batch so we don't overload services.